app/views/core/SubVueComponentView.js

```diff
@@ -0,0 +1,55 @@
+import CocoView from 'views/core/CocoView'
+import store from 'core/store'
+
+const silentStore = { commit: _.noop, dispatch: _.noop }
+
+export default class VueComponentView extends CocoView {
+  constructor (options) {
+    super(options)
+
+    if (!this.VueComponent) {
+      throw new Error('Class must be initialized with VueComponent')
+    }
+
+    this.props = {}
```

Comment:
Should we set some initial values to the state her in the constructor, just like in VueComponentView